templates/web/base/waste/bulky/items.html

```diff
@@ -9,6 +9,18 @@
   [% INCLUDE 'waste/_address_display.html' %]
 [% END %]
 
+<div class="no-js-message govuk-warning-text due" style="max-width:550px">
+  <div class="govuk-warning-text__img">
+    <span class="govuk-warning-text__icon" aria-hidden="true">i</span>
+  </div>
+  <div class="govuk-warning-text__content">
+    <span class="govuk-warning-text__assistive">Important information</span>
+    <p class="govuk-!-margin-bottom-1"><strong>About your items</strong></p>
+    <p >Before continuing, please read the following file and see if any items have a note or comment that might help our crew to collect them.</p>
+    <a class="btn-primary" href="" download="">Item notes</a>
```

Comment:
Could you just add an 'XXX' comment here explaining we will need a file?

Given that the item list is determined by the admin page (http://peterborough.localhost:3000/admin/waste/<id>/bulky_items) the file may need to be generated based on that, rather than provided by the client.

Reply:
@nephila-nacrea I don't have access to that url, so if you could generate it would be really helpful. I really don't have more time available this sprint, at the moment I'm jumping from one project to another and I have a heavy deadline for next week with climate.

Here is the ticket for the summary:
https://github.com/mysociety/societyworks/issues/3236
Not sure what has happened with it, if it has already been merged.

I just pushed a commit for the comment =)
Let me know if there is more feedback, but I'll probably won't be able to tackle it till next sprint.

Reply:
Sorry, I think I caused you confusion. I only really wanted the comment adde; I will approve the change now. The file-generation is something to be handled later 👍